### DISCUSSION TITLE
PaddleLabel加载自定义检测模型无法识别图片中目标区域

### DISCUSSION THREAD

**Discussion by gubinjie** [PaddlePaddle/PaddleOCR#12341] [Q&A]
PaddleOCR V2.7
1、使用DB++训练了检测模型，并将模型通过tools/export_model.py导出为inference模型。
2、在使用命令行：`python3 tools/infer/predict_det.py --det_algorithm="DB++" --det_model_dir="./output/yinzhang_det_240227/det_db_inference/" --image_dir="./train_data/yinzhang_240227/target/img/094_18.png" --use_gpu=True`检测时，能够检测图片中目标区域。
3、但是使用PaddleLabel加载导出来的模型时，一直无法识别目标区域。
4、启动命令及修改代码：
    `python ./PPOCRLabel.py --lang ch`，
    修改文件：PPOCRLabel.py
    ` self.ocr = PaddleOCR(use_pdserving=False,
                             use_angle_cls=True,
                             det=True,
                             cls=True,
                             use_gpu=gpu,
                             lang=lang,
                             show_log=True,
                             rec_image_shape="3, 160, 1600",
                             det_max_side_len=1600,
                             det_model_dir='D:\\JetBrains\\PycharmProjects\\PaddleOCR\\output\\yinzhang_det_240227\\det_db_inference')
        self.table_ocr = PPStructure(use_pdserving=False,
                                     use_gpu=gpu,
                                     lang=lang,
                                     layout=False,
                                     show_log=True,
                                     rec_image_shape="3, 160, 1600",
                                     det_max_side_len=1600,
                                     det_model_dir='D:\\JetBrains\\PycharmProjects\\PaddleOCR\\output\\yinzhang_det_240227\\det_db_inference')`
5、模型文件截图：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/37869445/346f005e-a88c-4875-99d7-dc28fe6d4354)
6、启动之后，在UI界面点击自动标注，但是并没有检测出目标区域

请问这是bug还是由于什么原因？

**Comment by gubinjie** [comment]
在使用DB++作为模型检测训练时，自定义模型启动的时候，paddleocr.py内SUPPORT_DET_MODEL参数会限制你只能使用DB模型，这种情况会不会有问题？

**Comment by Sencc** [comment]
我也是在PaddleLabel使用DB++模型就无法检测，请问你解决了吗？

**Comment by gubinjie** [comment]
@Sencc 在你PaddleLabel install之后的路径里面，找到paddleocr.py，修改SUPPORT_DET_MODEL参数，改成如图：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/37869445/1181df32-7d83-4122-a937-fe57d4688e31)

**Comment by Sencc** [comment]
@gubinjie 感谢分享
